### PR TITLE
Standardize setup.py

### PR DESCRIPTION
**Pull request by reinbach**
[ch10740]

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -31,7 +31,7 @@ extras =
     test
 deps =
      d21: django>=2.1,<2.2
-     d22: django>=2.2,<3.0
+     d22: django>=2.2,<2.3
```
Comment (domdinicola) [inline tox.ini]: next version would be 3.0, but like this is also fine

File: tests/test_admin.py [hunk 2]
```diff
@@ -1,9 +1,10 @@
-from django.test import SimpleTestCase
 from django.contrib.admin.sites import AdminSite
+from django.test import SimpleTestCase
```
Comment (domdinicola) [inline tests/test_admin.py]: maybe we can add isort mandatory for the build

Comment (reinbach) [reply tests/test_admin.py]: `isort` is run in CI

**Comment by domdinicola** [comment]
@saxix can you review this?

File: Makefile [hunk 3]
```diff
@@ -52,6 +52,8 @@ lint:
 requirements:
 	pipenv lock -r > src/requirements/install.pip
 	pipenv lock -r -d > src/requirements/testing.pip
+	sed -i "" 's/\(.*\)==.*/\1/g' src/requirements/install.pip && sed -i "" '1d' src/requirements/install.pip
```
Comment (robertavram) [inline Makefile]: can we comment what these lines do so that it can be read a little easier

File: setup.py [hunk 4]
```diff
@@ -2,31 +2,65 @@
...
+    content = []
     for f in files:
-        content += codecs.open(os.path.join(HERE, f), 'r').read()
-    return content
+        content.extend(codecs.open(os.path.join(ROOT, 'src', 'requirements', f), 'r').readlines())
+    return "\n".join(filter(lambda l:not l.startswith('-'), content))
+
+
+def check(cmd, filename):
+    out = subprocess.run(cmd, stdout=subprocess.PIPE)
```
Comment (robertavram) [inline setup.py]: I think we should add docstrings for all these helper functions

Comment (domdinicola) [reply setup.py]: I think we are gonna move to this PR https://github.com/unicef/unicef-vision/pull/7/files
I asked to use simple things

**Pull request closed without merging**